services/core-api/app/api/now_applications/models/now_application.py

```diff
@@ -188,6 +189,53 @@ class NOWApplication(Base, AuditMixin):
     def __repr__(self):
         return '<NOWApplication %r>' % self.now_application_guid
 
+    # @hybrid_property
```

Comment:
Can you guys please remove this commented-out code?


<small>Just kidding</small>